app/src/main/java/org/mozilla/tv/firefox/channels/DefaultChannelAdapter.kt

```diff
@@ -4,6 +4,8 @@
 
```

Comment:
Interesting note, it looks like this fixes a small visual bug.  I'm not sure what it's number was, but the STR are: have <= 4 tiles remaining in a row -> select any tile index 0 - 2 -> delete tile.  Tile at index 3 clips its border and looks weird.

Presumably one of the `noclip`s added in this PR fixed it.  Nice!